pandas/plotting/_core.py

```diff
@@ -2951,21 +2951,46 @@ def box(self, by=None, **kwds):
 
     def hist(self, by=None, bins=10, **kwds):
         """
-        Histogram
+        Draw one histogram of the DataFrame's Series using matplotlib.
```

Comment:
"the DataFrame's Series" -> "the DataFrame's columns" ?

The "using matplotlib" is not needed in this summary line, as all our plotting machinery is based on matplotlb